tests/datasets/test_cdl.py

```diff
@@ -31,9 +31,21 @@ def dataset(
         monkeypatch.setattr(  # type: ignore[attr-defined]
             torchgeo.datasets.cdl, "download_url", download_url
         )
+        cmap = {
```

Comment:
This cmap should be added directly to the file so we can test that TorchGeo can read the cmap from a file. See https://github.com/microsoft/torchgeo/tree/main/tests/data#raster-data for the code to add this to the file.